versioned_docs/version-8.5/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -313,7 +330,56 @@ tasklist:
 
 The actual configuration properties can be reviewed [in the Tasklist configuration documentation](/self-managed/tasklist-deployment/tasklist-configuration.md#zeebe-broker-connection).
 
-5. Configure all other applications running inside the cluster and connecting to the Zeebe Gateway to also use TLS.
+5. For Connectors:
+
+The Connectors component will only accept a `jks`(java keystore) certificate.
+If you have followed our previous recommendation of generating a TLS certificate using the OpenShift annotation, then you will have a `PKCS#1` certificate which the Connectors component will not accept. For convenience, here are a number of commands that will convert the `PKCS#1` certirficate, generated by OpenShift to `jks` format, that Connectors component accepts:
+
+```bash
+#grab OpenShift generated tls crt
+kubectl get secret -n camunda camunda-zeebe-gateway -o jsonpath="{.data['tls\.crt']}" | base64 --decode > tls.crt
+#grab OpenShift generated tls key
```

Comment:
All comments should be written properly.

e.g.

```
#grab OpenShift generated tls key
```
Should be:
```
# Grab OpenShift generated TLS key.
```

Reply:
✅

Reply:
@hamza-m-masood not fixed yet.

Reply:
✅